src/Layers/xrRenderDX10/dx10Texture.cpp

```diff
@@ -308,7 +308,7 @@ ID3DBaseTexture* CRender::texture_load(LPCSTR fRName, u32& ret_msize, bool bStag
     ZeroMemory(&IMG, sizeof(IMG));
 
     //  Staging control
-    static bool bAllowStaging = !strstr(Core.Params, "-no_staging");
+    static bool bAllowStaging = (RImplementation.o.no_ram_textures == false && !strstr(Core.Params, "-no_staging"));
```

Comment:
Now `-noramtex` and `-no_staging` became the same. So we can remove `-no_staging`